src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -231,12 +250,61 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
         const iframe = this.template.querySelector(
             `[data-id='${this.CUSTOM_LABELS.commonPaymentServices}']`);
 
-        //The cardholder name is always empty for the purchase Payments Services card tokenization iframe
-        //even though when it is accessible by the Gift Entry form for the Donor Type = Contact.
-        const nameOnCard = null;
-            return tokenHandler.requestToken(iframe, nameOnCard,
-                this.handleError, this.resolveToken)
+            return tokenHandler.requestToken({
+                iframe : iframe,
+                tokenizeParameters: this.buildTokenizeParameters(),
+                eventAction: this.tokenizeEventAction(),
+                handleError: this.handleError,
+                resolveToken: this.resolveToken
+            });
+
+    }
+
+    buildTokenizeParameters() {
+        if (this._currentPaymentMethod === CREDIT_CARD) {
+            //The cardholder name is always empty for the purchase Payments Services card tokenization iframe
+            //even though when it is accessible by the Gift Entry form for the Donor Type = Contact.
+            return { nameOnCard: null };
+        } else {
+            return this.ACHTokenizeParameters();
+        }
+    }
+
+    ACHTokenizeParameters() {
+        let achTokenizeParameters = {
+            nameOnAccount: '',
+            accountHolder: {}
+        };
+
+        if (this.accountHolderType() === ACCOUNT_HOLDER_TYPES.BUSINESS) {
+            achTokenizeParameters.accountHolder.businessName =
+                this.widgetDataFromState[apiNameFor(DATA_IMPORT_CONTACT_LASTNAME)];
+            achTokenizeParameters.accountHolder.accountName =
+                this.widgetDataFromState[apiNameFor(DATA_IMPORT_ACCOUNT_NAME)];
+            achTokenizeParameters.nameOnAccount =
+                this.widgetDataFromState[apiNameFor(DATA_IMPORT_ACCOUNT_NAME)];
```

Comment:
Do you think it'd be worth extracting this out? Maybe something like `populateAchParametersForBusiness(achTokenizeParameters)`?